jbpm-examples/pom.xml

```diff
@@ -94,6 +94,10 @@
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-test</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.jbpm</groupId>
+          <artifactId>jbpm-test-util</artifactId>
+      </dependency>
```

Comment:
Ok, it is needed since it is used in org/jbpm/examples/checklist/impl/CheckListUI.java but I was unable to find it since org.jbpm.test.util.PoolingDataSource is return type of JBPMHelper.setupDataSource() and the return value is ignored so it isn't used anywhere. But maven enforcer catches this as a direct usage. So this is OK.